### PR TITLE
New readme draft

### PR DESCRIPTION
**Pull request by jgthomas**
A first, simple re-draft of the readme page, submitted for feedback before going further.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,11 +1,39 @@
 What is borg-import?
 ====================
 
-borg-import can import backups from some other backup software, currently supported are:
+A set of scripts to import from other backup software into the archive format used by `BorgBackup <https://github.com/borgbackup/borg>`_.
```
Comment (ThomasWaldmann) [inline README.rst]: an `archive` (in borg terms) is the result of a `borg create` operation.

but there is also a special `repository` format (and only that is seen by the user when looking at a borg repo directly).

I don't think it is useful to talk about archives or repos here, maybe rather `storage format` or `backup format`. or just keep the previous phrasing.

`a set of scripts` also sounds worse than it is, maybe rather `a tool`?

File: README.rst [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
 
-- rsnapshot
-- ``rsynchl``: simple rsync+hardlinks (One folder per archive, using folder mtime)
-- Back In Time (through ``rsynchl``)
-- planned: complex rsync+hardlinks (e.g. Multiple hosts, separate backup time)
+See ``borg-import -h`` for general help.
 
-See ``borg-import -h`` for more information.
+Currently supported imports
```
Comment (ThomasWaldmann) [inline README.rst]: Currently supported import formats

File: README.rst [hunk 3]
```diff
@@ -1,11 +1,39 @@
...
+===========================
+
+`rsnapshot <https://github.com/rsnapshot/rsnapshot>`_
+---------
+
+Usage ``borg-import rsnapshot SNAPSHOT_ROOT BORG_REPOSITORY``.
```
Comment (ThomasWaldmann) [inline README.rst]: Usage:

and rather no dot at the end.

File: README.rst [hunk 4]
```diff
@@ -1,11 +1,39 @@
...
+
+See ``borg-import rsnapshot -h`` for help.
+
+Simple rsync with hard links
+----------------------------
+
+Assumes one folder per archive, using folder mtime
```
Comment (ThomasWaldmann) [inline README.rst]: Maybe have a complete sentence here, with a dot at the end.

File: README.rst [hunk 5]
```diff
@@ -1,11 +1,39 @@
...
+
+Usage ``borg-import rsynchl RSYNC_ROOT BORG_REPOSITORY``
+
+See ``borg-import rsynchl -h`` for help.
+
+Backup tools based on rsync with hard links
+-------------------------------------------
+
+Requires the tool match the assumptions listed for plain rsync with hard links
```
Comment (ThomasWaldmann) [inline README.rst]: same here

**Comment by jgthomas** [comment]
I have made all the changes suggested in your feedback

File: README.rst [hunk 6]
```diff
@@ -1,11 +1,41 @@
...
+
+Usage: ``borg-import rsynchl RSYNC_ROOT BORG_REPOSITORY``
+
+See ``borg-import rsynchl -h`` for help.
+
+Backup tools based on rsync with hard links
+-------------------------------------------
+
+borg-import should, in principle, be able to import backups from any backup tool that is
+based on rsync with hard links. This requires that the tool match the assumptions listed above for simple
```
Comment (ThomasWaldmann) [inline README.rst]: matches

**Comment by ThomasWaldmann** [comment]
thanks! merging...

**Pull request merged**